Formula/elan-init.rb

```diff
@@ -0,0 +1,51 @@
+class ElanInit < Formula
+  desc "Lean Theorem Prover installer and version manager"
+  homepage "https://leanprover-community.github.io/"
```

Comment:
```suggestion
  homepage "https://github.com/leanprover/elan"
```
A more specific homepage would be good here, I think.